rules/code-quality/src/Rector/Foreach_/SimplifyForeachToCoalescingRector.php

```diff
@@ -152,10 +152,11 @@ private function processForeachNodeWithReturnInside(Foreach_ $foreach, Return_ $
         } else {
             return null;
         }
+        $foreachNode = $foreach->getAttribute(AttributeKey::NEXT_NODE);
```

Comment:
Same princip as above:

```suggestion
        $nextNode = $foreach->getAttribute(AttributeKey::NEXT_NODE);
```

Reply:
implemented.